html/themes/custom/common_design_subtheme/templates/layout/region--sidebar-first.html.twig

```diff
@@ -0,0 +1,29 @@
+{#
+/**
+ * @file
+ * Theme override to display a region.
+ *
+ * Available variables:
+ * - content: The content for this region, typically blocks.
+ * - attributes: HTML attributes for the region <div>.
+ * - region: The name of the region variable as defined in the theme's
+ *   .info.yml file.
+ *
+ * @see template_preprocess_region()
+ */
+#}
+{%
+  set classes = [
+    'region',
+    'region-' ~ region|clean_class,
+    'cd-dropdown',
+  ]
+%}
+
+{{ attach_library('common_design/cd-dropdown') }}
+
+{% if content %}
+  <div {{ attributes.addClass(classes) }} role="menu" id="cd-filters-dropdown" aria-labelledby="cd-filters-dropdown-toggle" data-cd-toggable="Filters" data-cd-hidden="true" data-cd-icon="arrow-down" data-cd-component="cd-filters" data-cd-toggable-keep="true">
```

Comment:
@attiks What I would like to do here is have a condition that checks for something (maybe facet blocks?) and only then adds the markup/attributes which initiate the toggle that will display the contents of a dropdown. Currently, anything in the sidebar will be collapsed on mobile, and a "Filters" button will be created.
Each facet is a separate block and I want the whole thing to be hidden (not individual blocks) which is why I opted to do this in the sidebar region template. Can you recommend a way to check whether facets are present in the sidebar (or some other condition)  so I can control this better? Or if you have a better idea how to approach this.
This is what is should result in:
![sidebar-with-toggle](https://user-images.githubusercontent.com/1835923/79850927-e3638400-83c4-11ea-98c5-a7aaf02582d9.png)
This works right now but it will affect the sidebar regardless of whether there are facet blocks.

Reply:
Added, see https://github.com/UN-OCHA/assessmentregistry8-site/pull/50/commits/eba610171e05ed09a9997c71c446b3cd0ce8ba05